lib/Munin/Master/UpdateWorker.pm

```diff
@@ -1147,13 +1147,18 @@ sub _create_rrd_file {
               "RRA:AVERAGE:0.5:288:450", # 450 days, resolution 1 day
               "RRA:MIN:0.5:288:450",
               "RRA:MAX:0.5:288:450");
-    }
-    elsif ($resolution eq 'huge') {
+    } elsif ($resolution eq 'huge') {
```

Comment:
I am relieved, that you do not seem to feel attached to the multi-line `} elsif` style :)